test/build.test.js

```diff
@@ -19,6 +19,8 @@ if (process.platform === 'win32') {
 process.env.PATH = cmd_path + sep + process.env.PATH;
 process.env.NODE_PATH = path.join(__dirname,'../lib/');
 
+var jsEngine = process.jsEngine ? '-' + process.jsEngine : '';
```

Comment:
It was in order to not to create a breaking change. If we add the suffix `v8` everywhere it could break the versioning of the modules (such as `fsevents`)